# UGE-MPC controller configuration
# This file provides parameters consumed by `uge_controller_node.py`.


T: 30 # Horizon time steps ~ 4 seconds

control_dt: 0.2

step_val: 10 # number of ellipsoid skips

# v_bounds: [0.1, 3.0]

# delta_bounds: [-0.45, 0.45] # in rad ~ 20 degrees

Sigma0: [0.01, 0.01, 1.0] # in degrees

Q: [0.5, 2.0] # in degrees

R: [1.0, 10.0] # in degrees

num_control_rollouts: 256 

rec_min_control_rollouts: 100

vehicle:
  length: 0.78
  wheelbase: 0.48
  width: 0.54
  vehicle_type: "kinematic"
  v_bounds: [0.1, 1.5]
  delta_bounds: [-0.53, 0.53] # in rad ~ 20 degrees


